src/test/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricPerformanceJudgeTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.EDIT_ISSUE
+import com.atlassian.performance.tools.report.api.ShiftedDistributionRegressionTest
+import com.atlassian.performance.tools.report.api.result.DurationData
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+import com.atlassian.performance.tools.report.junit.FailedActionJunitReport
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.mockito.Mockito
```

Comment:
Let's avoid Mockito or and all reflection-based undebuggable libs.

Reply:
I think this discussion is outside of the PR scope. I'd like to know the detailed rationale behind avoiding mockito as we might just have a different experience in this area